diff --git a/node_modules/tui-editor/dist/tui-editor-Editor.js b/node_modules/tui-editor/dist/tui-editor-Editor.js
index 91cceff..23149a5 100644
--- a/node_modules/tui-editor/dist/tui-editor-Editor.js
+++ b/node_modules/tui-editor/dist/tui-editor-Editor.js
@@ -19613,7 +19613,7 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
  * @type {string}
  * @ignore
  */
-var containerTmpl = ['<div class="tui-editor">', '<div class="te-md-container">', '<div class="te-editor" />', '<div class="te-md-splitter" />', '<div class="te-preview" />', '</div>', '<div class="te-ww-container">', '<div class="te-editor" />', '</div>', '</div>'].join('');
+var containerTmpl = ['<div class="tui-editor">', '<div class="te-md-container">', '<div class="te-editor"></div>', '<div class="te-md-splitter"></div>', '<div class="te-preview"></div>', '</div>', '<div class="te-ww-container">', '<div class="te-editor"></div>', '</div>', '</div>'].join('');
 
 /**
  * Class Layout
